src/inject/gradeDist.js

```diff
@@ -73,7 +75,6 @@ GradeDist.prototype.getNewData = function (teacher, course, callback) {
         var abort = false;
         $(tableRows).each(function (i, tr) {
             var cells = $(tr).children();
-
             if (cells.length <= 11) {
```

Comment:
Could you update this line to check against 13 rather than 11 since the code will now reference up to index 13 of `cells`.